src/libraries/packages.builds

```diff
@@ -70,4 +70,14 @@
   </Target>
 
   <Import Project="$(MSBuildThisFileDirectory)dir.traversal.targets" />
+
+  <Target Name="SetAzureDevOpsVariableForBuiltPackages"
+          Condition="'$(OfficialBuildId)' != ''"
```

Comment:
Actually this also needs to run on CI as installer needs produced packages to be in the bin output. Will update.